openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/ProfilerDashboard.tsx

```diff
@@ -0,0 +1,353 @@
+      );
+    } catch (error) {
+      showErrorToast(
+        error as AxiosError,
+        jsonData['api-error-messages']['update-entity-unfollow-error']
+      );
+    }
+  };
+  const followTable = async () => {
+    try {
+      const data = await addFollower(table.id, getCurrentUserId());
+      const { newValue } = data.changeDescription.fieldsAdded[0];
+
+      setFollower([...follower, ...newValue]);
+    } catch (error) {
+      showErrorToast(
+        error as AxiosError,
+        jsonData['api-error-messages']['update-entity-follow-error']
+      );
+    }
+  };
+
+  const handleFollowClick = () => {
+    if (isFollowing) {
+      setIsFollowing(false);
+      unfollowTable();
+    } else {
+      setIsFollowing(true);
+      followTable();
+    }
+  };
+
+  const handleTabChange = (e: RadioChangeEvent) => {
+    const value = e.target.value as ProfilerDashboardTab;
+    if (ProfilerDashboardTab.SUMMARY === value) {
+      history.push(getTableTabPath(table.fullyQualifiedName || '', 'profiler'));
+    }
+    setActiveTab(value);
+  };
+
+  const handleAddTestClick = () => {
+    history.push(
+      getTableTabPath(table.fullyQualifiedName || '', 'data-quality')
+    );
+  };
+
+  const handleTimeRangeChange = (value: keyof typeof PROFILER_FILTER_RANGE) => {
+    if (value !== selectedTimeRange) {
+      setSelectedTimeRange(value);
+      fetchProfilerData(table.id, PROFILER_FILTER_RANGE[value].days);
+    }
+  };
+
+  useEffect(() => {
+    if (table) {
+      const columnName = getNameFromFQN(entityTypeFQN);
+      const selectedColumn = table.columns.find(
+        (col) => col.name === columnName
+      );
+      setActiveColumnDetails(selectedColumn || ({} as Column));
+      setFollower(table?.followers || []);
+      setIsFollowing(
+        follower.some(({ id }: { id: string }) => id === getCurrentUserId())
+      );
+    }
+  }, [table]);
+
+  return (
+    <PageLayout>
+      <Row gutter={[16, 16]}>
+        <Col span={24}>
+          <EntityPageInfo
+            isTagEditable
+            deleted={table.deleted}
+            entityFqn={table.fullyQualifiedName}
+            entityId={table.id}
+            entityName={table.name}
+            entityType={EntityType.TABLE}
+            extraInfo={extraInfo}
+            followHandler={handleFollowClick}
+            followers={follower.length}
+            followersList={follower}
+            hasEditAccess={hasEditAccess(
+              table.owner?.type || '',
+              table.owner?.id || ''
+            )}
+            isFollowing={isFollowing}
+            tags={getTagsWithoutTier(table.tags || [])}
+            tagsHandler={onTagUpdate}
+            tier={tier}
+            titleLinks={breadcrumb}
+            updateOwner={onOwnerUpdate}
+            updateTier={onTierUpdate}
+          />
+        </Col>
+        <Col span={24}>
+          <Row justify="space-between">
+            <Radio.Group
```

Comment:
Can we use antd tab component here?
https://ant.design/components/tabs/

Reply:
I ll handle this in my next PR as this feature is still work in progress